### DISCUSSION TITLE
Time always shows one hour early

### DISCUSSION THREAD

**Discussion by sohnavcom** [roubachof/MetroLog#42] [Q&A] [answered]
Hi the time always seems to show 1 hour early. Im here in the Uk. I am already using a custom layout

`  public class MetroLogLayout : Layout
  {

      public override string GetFormattedString(LogWriteContext context, LogEventInfo info)
      {
          return $"{info.TimeStamp:G} - {info.Level}: {info.Message}";
      }
  }

Any ideas?`

**Comment by roubachof** [comment, marked as answer]
Time is UTC

**Comment by sohnavcom** [comment]
ok thanks
